compiler/rustc_attr_parsing/src/context.rs

```diff
@@ -255,13 +255,16 @@ impl Stage for Early {
             sess.dcx().create_err(diag).delay_as_bug()
         }
     }
+
+    fn should_emit(&self) -> ShouldEmit {
+        self.emit_errors
```

Comment:
Note that `Early` already contained an `emit_errors`, and `Late` should always emit errors, so this assoc const is a little useless

Reply:
I agree